common/wonders/00_wonders.txt

```diff
@@ -804,11 +804,18 @@ wonder_statue_ruler = {
 	picture = GFX_wonder_strip_ruler_statue
 	model = wonder_statue
 	potential = {
+		# Warcraft
```

Comment:
Why are great statues banned for evil religions? Surely Demon/Old God minions have erected monuments in honor of themselves?

Reply:
Statues are fine, but there's a lot of upgrades that don't look good with evil guys. Like tavern, statues of family (even if you don't have it), hypocausts, heating pipes.